client/components/map/mapToolTip.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { Popup } from 'react-leaflet'
-const Tooltip = ({distance, trailName, ...props}) => (
+const Tooltip = ({distance, trailName, isActive, ...props}) => (
```

Comment:
[this change](https://github.com/NathanBland/trails/pull/11/files#diff-91bcfdd0a2ac0af8b737e0ea94a73545R49) disagrees with you